src/lib/PlaylistCards.svelte

```diff
@@ -11,19 +11,44 @@
 
   export let playlists: Playlist[] = []
 
+  const rawCurationOverrides = import.meta.glob(
+    '/src/collections/playlist-curation-date-overwrites/*.md',
+    { eager: true },
+  )
+
+  let curationOverrideMap: Record<string, string> = {}
+  for (const path in rawCurationOverrides) {
+    const file = rawCurationOverrides[path]
+    const { id, curationOverwrite } = file.metadata || file
```

Comment:
Not sure what the eventuality is that is being covered here, but it shouldnt break anything